src/uct/ib/ud/accel/ud_mlx5.c

```diff
@@ -405,9 +429,11 @@ uct_ud_mlx5_ep_create_connected(uct_iface_h iface_h,
     ucs_trace_data("TX: CREQ (qp=%x lid=%d)", if_addr->qp_num, if_addr->lid);
     uct_ud_mlx5_ep_tx_skb(iface, ep, skb);
     uct_ud_iface_complete_tx_skb(&iface->super, &ep->super, skb);
+    uct_ud_leave(&iface->super);
     return UCS_OK;
 
 err:
+    uct_ud_leave(&iface->super);
```

Comment:
looking at order of enter/create, do we need to reverse order of leave/destroy here?